<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="output" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>90</y>
      <width>761</width>
      <height>450</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="text_prase_putout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>450</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="input_verilog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>../../src/netlist2schematic/input/one_full_adder.v</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prase">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Prase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_skin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>../../src/netlist2schematic/conf/rsfq.svg</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
